benefits/eligibility/templates/eligibility/confirm.html

```diff
@@ -26,7 +26,8 @@ <h1>{{ form.headline }}</h1>
 {% endblock explanatory-text-wrapper %}
 
 {% block inner-content %}
-  <div class="container">
-    <div class="row">{% include "core/includes/form.html" with form=form %}</div>
-  </div>
+  {% include "core/includes/form.html" with form=form %}
 {% endblock inner-content %}
+
+{% block call-to-action %}
+{% endblock call-to-action %}
```

Comment:
Why is this here?

It's here to ensure that more spacing is not created. If this is _not_ here, then the `base.html` will put this HTML on the page, which includes `pt-8` -- which we do *not* want: 

```html
            <div class="row d-flex justify-content-center pt-8">
              <div class="col-12 col-lg-6">
                {% block call-to-action-button %}
                {% endblock call-to-action-button %}
              </div>
            </div>
```

There's an alternate way to achieve this. I could have put this code on both `eligibility/confirm.html` and `eligibility/index.html`:

```html
                {% block call-to-action-button %}
                    {% include "core/includes/recaptcha-text.html" %}
                {% endblock call-to-action-button %}
```

But it felt semantically incorrect to put _text_ into code that was meant to be for a button only. And I didn't want to come up with a more generic name to rename `call-to-action-button` or rename all those on the other templates... So I went with this method instead.

Reply:
This sounds reasonable :+1: I think a comment above the empty block in this file just indicating that it is there for a reason (maybe a link to this thread here?) would be helpful for the future.

Reply:
Added comments in both templates to clarify what is going on there